.vscode/settings.json

```diff
@@ -2,5 +2,11 @@
   "cSpell.words": [
     "kusto",
     "logmining"
-  ]
+  ],
+  "markdownlint.config": {
```

Comment:
This is not necessarily for the template

Reply:
Agree.